src/Compilers/CSharp/Test/Semantic/Semantics/UnsafeTests.cs

```diff
@@ -9314,5 +9314,495 @@ public unsafe struct CAPROPVARIANT
             var comp = CreateCompilation(csharp, options: TestOptions.UnsafeDebugDll);
             comp.VerifyDiagnostics();
         }
+
+        [Fact]
+        public void TestUnsafeAlias1()
```

Comment:
Please add a similar test but with regular (as opposed to unsafe) compilation options. We should get a diagnostic on `using unsafe ...`.

Reply:
Done.